ihmc-high-level-behaviors/src/main/java/us/ihmc/perception/RapidHeightMapManager.java

```diff
@@ -108,8 +110,8 @@ public void update(Mat latestDepthImage,
 
    public HeightMapData getLatestHeightMapData()
    {
-      HeightMapData temp = new HeightMapData((float) RapidHeightMapExtractorCUDA.getHeightMapParameters().getGlobalCellSizeInMeters(),
-                                             (float) RapidHeightMapExtractorCUDA.getHeightMapParameters().getGlobalWidthInMeters(),
+      HeightMapData temp = new HeightMapData((float) RapidHeightMapExtractor.getHeightMapParameters().getGlobalCellSizeInMeters(),
+                                             (float) RapidHeightMapExtractor.getHeightMapParameters().getGlobalWidthInMeters(),
```

Comment:
Doesn't this change break the whole thing right now? I suppose it's alright since it's still WIP